### DISCUSSION TITLE
Where is the changelog?

### DISCUSSION THREAD

**Discussion by tembbo** [nuxt/nuxt#28250] [General]
Shouldn't there be a CHANGELOG.md in the repo, or is it just me that can't find it.

**Comment by xjccc** [comment]
[look at this for some details](https://github.com/nuxt/nuxt/releases/tag/v3.12.4)

**Comment by tembbo** [comment]
yeah, i looked at that, but shouldn't there be a file (CHANGELOG.md) with all the changelogs from the releases
